### PR TITLE
feat(stack): use Animated.Text for header title

### PR DESCRIPTION
**Pull request by dulmandakh**
This PR changes HeaderTitle to use Animated.Text, so that developers can use Animated values, like opacity, in headerTitleStyle.

### CODE REVIEW

**Comment by dulmandakh** [comment]
IIRC, react navigation 4.x and below supported Animated values in headerTitleStyle.

**Comment by dulmandakh** [comment]
I use it to show title if a content is scrolled to a certain point

**Comment by dulmandakh** [comment]
@satya164 please review and merge

File: packages/stack/src/views/Header/HeaderTitle.tsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-import { Text, StyleSheet, Platform, TextProps } from 'react-native';
+import { Animated, StyleSheet, Platform, TextProps } from 'react-native';
 
 type Props = TextProps & {
   children?: string;
 };
 
 export default function HeaderTitle({ style, ...rest }: Props) {
-  return <Text {...rest} style={[styles.title, style]} />;
+  return <Animated.Text {...rest} style={[styles.title, style]} />;
```
Comment (satya164) [inline packages/stack/src/views/Header/HeaderTitle.tsx:10]: We're using Reanimated in stack. So we should change this to import `Animated` from `react-native-reanimated` instead.

**Review by satya164 — CHANGES_REQUESTED** [review]

File: packages/stack/src/views/Header/HeaderTitle.tsx (hunk 1 above)
Comment (dulmandakh) [reply packages/stack/src/views/Header/HeaderTitle.tsx:10]: done 👍

**Review by satya164 — APPROVED** [review]

**Pull request merged**